src/index.js

```diff
@@ -11,10 +11,13 @@ import { router as photonRouter } from './photon/index.js';
 import { router as nominatimRouter } from './nominatim/index.js';
 import { router as fileRouter } from './file/index.js';
 
+if (!process.version.startsWith('v18.')) {
```

Comment:
I dont really like this but I think I get why you want it. id rather this be gone or at the very least try to match `>=18` since node versions change so rapidly.